client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Connection.spec.ts

```diff
@@ -112,10 +112,10 @@ describe("Connection", () => {
             }
         } as IHttpConnectionOptions;
 
-        var connection = new HttpConnection("http://tempuri.org", options);
+        let connection = new HttpConnection("http://tempuri.org", options);
```

Comment:
I don't think so - this is not an object literal so I don't think it makes any difference. Conceptually this object is not a const since it exposes `onClosed` and `onDataReceived` properties which are settable.